### PR TITLE
Readme description of the project

### PR DESCRIPTION
**Pull request by petr-tik**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+The website for the Foreign Function Interface generator. 
+
+http://ffig.org/
```
Comment (ajbennieston) [inline README.md]: Maybe move this right to the top?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+The website for the Foreign Function Interface generator.
```
Comment (ajbennieston) [inline README.md]: If the URL is moved to the top, this line can go away.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+The website for the Foreign Function Interface generator. 
+
+http://ffig.org/
+
+A front-end for the FFIG CLI tool, which generates wrappers in high-level languages for Cpp header files.
+
+Insert your header file and choose, which bindings you want to generate
```
Comment (ajbennieston) [inline README.md]: No need for a comma.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+The website for the Foreign Function Interface generator. 
+
+http://ffig.org/
+
+A front-end for the FFIG CLI tool, which generates wrappers in high-level languages for Cpp header files.
```
Comment (ajbennieston) [inline README.md]: "FFIG generates high-level language wrappers for C++ classes and functions"

**Review by ajbennieston — APPROVED** [review]

**Comment by petr-tik** [comment]
please merge, I don't have write access

**Comment by ajbennieston** [comment]
Neither do I

**Review by jbcoe — APPROVED** [review]

**Comment by jbcoe** [comment]
Should be mergeable now.

**Pull request merged**